ang/civicase/case/details/people-tab/directives/case-details-people-tab.directive.js

```diff
@@ -421,6 +437,19 @@
         ['CaseContact', 'create', {
           case_id: item.id,
           contact_id: contactPromptResult.contact.id
+        }],
+        ['Relationship', 'get', {
+          case_id: item.id,
+          is_active: 1,
+          'api.Relationship.create': {
+            id: false,
+            contact_id_a: contactPromptResult.contact.id,
+            start_date: 'now',
+            contact_id_b: '$value.contact_id_b',
+            relationship_type_id: '$value.relationship_type_id',
+            description: '$value.description',
+            case_id: '$value.case_id'
+          }
         }]
```

Comment:
Same comment as 🍰 

It is difficult to follow all the things happening in this function and chained api calls don't make it easier.